packages/rocketchat-lib/server/methods/sendMessage.js

```diff
@@ -10,6 +10,10 @@ Meteor.methods({
 			});
 		}
 
+		if (!message.rid) {
+			throw new Error('The \'rid\' param, must be provided');
```

Comment:
Maybe say `The 'rid' property on the message object is missing.`? Does that make more sense?